app/react/Metadata/helpers/groupRelationshipFields.ts

```diff
@@ -0,0 +1,31 @@
+import { PropertySchema } from 'shared/types/commonTypes';
+
+export function groupSameRelationshipFields(fields: PropertySchema[]) {
```

Comment:
This function looks like it's the same implemented in `MetadataFormFields` and it's not used anywhere. Can you please take a look?

Reply:
yep, deleted it, it was for the previous approach of the form